src/ontology/reports/pato-edit.obo-obo-report.tsv

```diff
@@ -1,6 +1,8 @@
 Level	Rule Name	Subject	Property	Value
-WARN	missing_definition	part of	definition	
-WARN	missing_definition	has part	definition	
+ERROR	missing_label	CL:0000000	rdfs:label	
```

Comment:
that is odd... How did that happen? Can we increase the failure level to fail on ERROR @shawntanzk and check whats up with that label?

Reply:
probably an important that was not refreshed..